src/main/java/nextstep/converter/FavoriteConverter.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.converter;
+
+import nextstep.favorite.application.dto.FavoriteResponse;
+import nextstep.favorite.domain.Favorite;
+import nextstep.station.dto.StationResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static nextstep.converter.StationConverter.stationToStationResponse;
+
+public class FavoriteConverter {
+
+    private FavoriteConverter() {
+        throw new UnsupportedOperationException("Utility class");
+    }
+
+    public static List<FavoriteResponse> favoriteListToFavoriteResponseList(final List<Favorite> favoriteList) {
+        List<FavoriteResponse> favoriteResponseList = new ArrayList<>();
```

Comment:
Collection 자료형의 변수 이름에 `-List`와 같은 자료형을 포함하는 대신에, 복수형으로 지어보면 어떨까요?
https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming/
```suggestion
    public static List<FavoriteResponse> favoriteListToFavoriteResponseList(final List<Favorite> favorites) {
        List<FavoriteResponse> favoriteResponses = new ArrayList<>();
```